test/kubemark/run-e2e-tests.sh

```diff
@@ -14,16 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBERNETES_PROVIDER="kubemark"
+# TODO(shyamjvs): This variable should be read from a cloud_provider_config.sh file.
```

Comment:
It's not possible to comment on the previous file in github, so commenting here.

The only gce-specific command in configure-kubectl.sh is where do we get kubectl from. The main part that is really configuring kubectl is provider-independent. So we should either:
- split this into two scripts (install kubectl and configure it)
- or assume that kubectl is in PATH and remove the first part.

I guess for now, the former is easier.